RELEASING.md

```diff
@@ -0,0 +1,32 @@
+* Choose the right version number:
+  * `x.0.0` is reserved for incompatible changes and require a milestone cycle.
+  * `x.y.0` is reserved for compatible changes.
+  * `x.y.z` is reserved for bugfixes that don't change the public API or the SemanticDB schema.
+  * For more details, consult [versioning policy](VERSIONING.md).
+* Tag the release:
+  * The tag must be called `vx.y.x`, e.g. `v3.0.0`.
+  * To create the tag:
+    * https://github.com/scalameta/scalameta/releases/new
+    * Or, `git tag -a vx.y.z -m "vx.y.z"` && `git push upstream --tags`
+* Close the milestone corresponding to the release.
+* Wait for [the Travis CI job](https://travis-ci.org/scalameta/scalameta/branches)
+   in Active Branches to build the binaries and stage them to Sonatype.
+* Finalize the release on Sonatype:
```

Comment:
Alternatively, `sbt sonatypeReleaseAll` from the scalameta build.

I would love to get rid of this step by adding a build stage that does this step after all release jobs complete.